004-governance.Rmd

```diff
@@ -0,0 +1,153 @@
+We do have a basic set of assumptions under which we operate, but we have not previously made them precise.
+This tidyup lays out a model that strikes a balance between the benevolent dictator and rough consensus models, and defines the relationships between four key roles: a large community of package **users,** a smaller pool of GitHub **contributors**, a team of **authors**, and a single **maintainer**.
+
+## Motivation
+
+We currently have one package (ggplot2) with a [formal governance model](https://github.com/tidyverse/ggplot2/blob/master/GOVERNANCE.md), and an informal, undocumented model that we've used for other packages (e.g. dtplyr, dbplyr).
+The goal of this tidyup is to come up with holistic, flexible model, that can become our default governance model going forward.
+This is part of a general movement to get better at defining and documenting our processes to so that there's a clear path from package user to package developer.
+
+## Solution
+
+The following sections describes a governance model to use as a default for open source RStudio repositories (starting with those in the r-lib, tidyverse, and tidymodels organisations).
+It is not mandatory, but it has been designed to reflect our current best practices, and should be used unless there are compelling reasons to favour a different approach.
+
+The model strikes a balance between the [benevolent dictator](http://oss-watch.ac.uk/resources/benevolentdictatorgovernancemodel) and [rough consensus](https://datatracker.ietf.org/doc/html/rfc7282) governance models.
+We blend these two governance models because we want to involve the community in decision making as much as possible, while recognising that excellent user experience often requires consistent design guided by a single voice.
+
+### Users
+
+People who use the package are the most important members of the community; without these users, this project would have no purpose.
+Users are encouraged to participate in the life of the project and the community as much as possible.
+Common user activities include (but are not limited to):
+
+-   Evangelising about the project.
+-   Asking and answering questions on community forums.
+-   Providing moral support (a "thank you" goes a long way).
+
+### Contributors
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved.
+Such users may then go on to become **contributors** by interacting with the project on GitHub.
+Contributors:
+
+-   Report bugs and suggest improvements by creating new issues.
+
+-   Improving existing issues by answering questions, creating reprexes, or providing feedback on proposed changes.
+
+-   Contributing code or documentation via pull requests.
+
+Anyone can become a contributor: there is no expectation of commitment to the project, no required set of skills, and no selection process.
+The only requirement is to follow the code of conduct.
```

Comment:
Do we have a link to a standard code of conduct we use?

Reply:
Yes, this is the standard one. https://github.com/tidyverse/ggplot2/blob/master/CODE_OF_CONDUCT.md Working on some updates to it too.

Reply:
I'll leave the CoC generic in this doc, and we can fill in later.